server/services/PolicyService.ts

```diff
@@ -181,33 +181,27 @@ export default class PolicyService {
         "policy.policy.description": "policy.description.keyword",
         "policy.policy.last_updated_time": "policy.last_updated_time",
       };
+
       const params = {
-        index: INDEX.OPENDISTRO_ISM_CONFIG,
-        seq_no_primary_term: true,
-        body: {
-          size,
-          from,
-          sort: policySorts[sortField] ? [{ [policySorts[sortField]]: sortDirection }] : [],
-          query: {
-            bool: {
-              filter: [{ exists: { field: "policy" } }],
-              must: getMustQuery("policy.policy_id", search),
-            },
-          },
-        },
+        from,
+        size,
+        sortOrder: sortDirection,
+        sortField: policySorts[sortField] || policySorts.id,
+        queryString: search.trim() ? `*${search.trim().split(" ").join("* *")}*` : "*",
```

Comment:
Do you need all these "*<search>*<term>*" wildcards? I'd expect the backend API handles it now?

Reply:
Backends only take care if queryString is empty
This logic: split by " " and join with `* *` is not in backend
so we can just keep like this